DangoPlop/Assets/Scripts/PowerupMaster.cs

```diff
@@ -199,5 +211,21 @@ public class PowerupMaster : MonoBehaviour {
 		Vector3 originalPosition = playerObject.transform.position;
 	}
 
+	private void ResetBulletType(){
+		playerController.BulletType = "DefaultFire";
+	}
+
+	private void DoubleShot(){
+		playerController.BulletType = "DoubleShot";
+	}
+
+	private void Laser(){
+		playerController.BulletType = "Laser";
+	}
+
+	private void RapidFire(){
+		playerController.Ammo = 5;
```

Comment:
Let's just make the player have infinite ammo with the normal fire rate for rapid fire.